### ISSUE TITLE
Make clear why ownership matters in PI guide

### ISSUE THREAD

**Issue by nambrot** [hyperlane-xyz/hyperlane-monorepo#1861]
- make clear what contracts are owned
- consider making the deployer the owner and have owner configuration be optional

**Comment by asaj** [comment]
> consider making the deployer the owner and have owner configuration be optional

I like having an explicit owner? Generally not best practices to have your contracts owned by an EOA, and further more our tooling/docs encourage deploying with an insecure EOA at that.

**Comment by nambrot** [comment]
Yeah don't feel strongly about it, was just a suggestion to consider. Personally, for ownership I find it okay to be owned by the deployer first and then relinquish control later. Generally, I would optimize for reducing the surface area of configuration to get to success quicker if we can and have deployers do things after the initial success point. In any case, up whoever works on this.

**Comment by nambrot** [comment]
fwiw, the OZ erc20 wizard i believes makes the deployer the owner. Most ownable contracts default to msg.sender as the owner

**Comment by nambrot** [comment]
Folks keep asking

https://discord.com/channels/554623348622098432/1006683697841459230/1088502245068390450
https://discord.com/channels/935678348330434570/961710804011458621/1088536872239763456
